data/json/overmap_specials.json

```diff
@@ -1443,6 +1443,25 @@
         "unique" : false, "//":"only allow one per city",
         "required" : false, "//":"is this special required for valid city sizes",
         "flags" : [ "CLASSIC" ]
+    },
+	{
+        "type" : "overmap_special",
+        "id" : "Home Improvement Superstore",
+        "overmaps" :
+        [
+            { "point":[0,1,0], "overmap": "hdwr_large_entrance", "connect":"road"},
+            { "point":[-1,1,0], "overmap": "hdwr_large_SW"},
+            { "point":[-1,0,0], "overmap": "hdwr_large_NW"},
+            { "point":[0,0,0], "overmap": "hdwr_large_NE"}
+        ],
+        "locations" : [ "wilderness" ], "//":"what special locations does it spawn",
+        "city_distance" : [1, 10], "//":"how far from a city it should be",
+        "city_sizes" : [4, 20], "//":"what city sizes should it spawn in",
+        "occurrences" : [10, 20], "//":"how many per overmap",
```

Comment:
I figured it was something along those lines, but I just wanted to make sure because we've had problems with specials with high minimum occurrences stepping on other specials in the past.